agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/mustering-game-state/player-mustering-game-state/PlayerMusteringGameState.ts

```diff
@@ -221,14 +213,22 @@ export default class PlayerMusteringGameState extends GameState<ParentGameState>
         });
     }
 
-    getPotentialGainedPowerTokens(region: Region): number {
+    consolidatePowerOrderPresent(region: Region | null): boolean {
```

Comment:
It's weird that this function accept a `null` parameter. On top of that, the name should be `isConsolidatePowerOrderPresent`.

Reply:
I allowed the null parameter to make it easier in the component to use it where selectedRegion can be null. Shall I still remove the null?

Reply:
Yeah, if the component has a possibly `null` property, check before or cast the variable to `Region` when calling the function.